pertpy/tools/_metadata/_cell_line.py

```diff
@@ -78,217 +90,174 @@ def __init__(self):
             )
             _download(
                 url="https://www.intogen.org/download?file=IntOGen-Drivers-20200201.zip",
-                output_file_name=driver_gene_intOGen_file_name,
+                output_file_name="IntOGen-Drivers.zip",
                 output_path=settings.cachedir,
                 is_zip=True,
             )
+            os.remove(settings.cachedir.__str__() + "/IntOGen-Drivers.zip")
```

Comment:
Should we remove this? When this function is triggered again, it will now try to download it again, right? That's unnecessary.